services/tenant-ui/frontend/src/store/reservationStore.ts

```diff
@@ -87,10 +83,22 @@ export const useReservationStore = defineStore('reservation', () => {
     error.value = null;
     loading.value = true;
 
-    await api
-      .get(API_PATH.MULTITENANCY_RESERVATION(reservationId))
+    /**
+     * The status code of 404 needs to be propagated to the UI.
+     * Axios currently has a bug that doesn't allow error status codes
+     * to be passed to the "catch" block. It appears this only happens when
+     * Axios is preconfigured with the "create" method.
+     *
+     * This is a workaround to allow the 404 to be passed to the UI.
+     */
+    await axios(API_PATH.MULTITENANCY_RESERVATION(reservationId), {
+      method: 'GET',
+      // Allow 404 to still approve the promise
+      validateStatus: (stat: number) => stat === 404,
+    })
       .then((res) => {
```

Comment:
This is just 404ing everything since it's trying to request to the frontend app

Reply:
Why do you think that? It's using the same url as the rest of the store.

Reply:
I can do some more testing of different scenarios.

Reply:
Can you give me an example @loneil  where it's not working?

Reply:
Try to do any check status on any valid reservation
![image](https://user-images.githubusercontent.com/17445138/217074965-4e59f9c0-8e7f-49c1-b56f-df4fd2917094.png)

Reply:
It's taken out the axios instance that sets it to call the configured API endpoint so that will 404 anything.

Reply:
K. Thanks. I'll have a look.

Reply:
Looks like we're back to square one here. I'll have to look at this being an issue with the proxy server.